### ISSUE TITLE
Responsive tables don't scroll horizontally on iOS in Arabic (dir="RTL") template.

### ISSUE THREAD

**Issue by aaornrylow** [wet-boew/wet-boew#7368]
Hi,

The responsive tables work fine in a browser and on Android phone, but on iOS they won't scroll horizontally and are therefore unreadable. I've tested this on an iPhone 5 and iPhone 6.

I can confirm that the cause of the bug is the (dir="RTL") attribute on the page, as it contains both Arabic and English tables but only the English ones (wrapped in "LTR" divs) scroll properly.

Can you recommend a workaround? Any assistance you can provide would be greatly appreciated!

Thanks,

-Aaron

**Comment by hsrudnicki** [comment]
Is this still an issue? Please update the status of this, whether this can be closed or otherwise, thank you

**Comment by aaornrylow** [comment]
Yes it is. I can confirm that the scrolling still doesn't work on iOS.
